components/network/openconnect/Makefile

```diff
@@ -0,0 +1,77 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.illumos.org/license/CDDL.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright 2016 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		openconnect
+# Commit timestamp and previous known release
+COMPONENT_VERSION=	7.7.20161105
+COMPONENT_VERSION_TAG=	b8d3971124b5b2ecccaf727ecc48ba94b531df8e
+COMPONENT_VERSION_TAGSHORT=	b8d3971
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION_TAG).tar.gz
```

Comment:
Please, rename COMPONENT_ARCHIVE to $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
(your COMPONENT_ARCHIVE_URL will still reference $(COMPONENT_VERSION_TAG).tar.gz